src/Microsoft.VisualStudio.Services.Agent/JobServerQueue.cs

```diff
@@ -76,7 +76,7 @@ public sealed class JobServerQueue : AgentService, IJobServerQueue
         // In this way, customer still can get instance live console output on job start,
         // at the same time we can cut the load to server after the build run for more than 60s
         private int _webConsoleLineAggressiveDequeueCount = 0;
-        private int _webConsoleLineUpdateRate = _delayForWebConsoleLineDequeueDefault.Milliseconds;
+        private int _webConsoleLineUpdateRate = (int) _delayForWebConsoleLineDequeueDefault.TotalMilliseconds;
```

Comment:
This was the actual bug. Milliseconds was "0" so the updates would happen every "moment"

Reply:
Sigh, great catch!